HW1 Szymon Tworkowski

syzymon opened the pull request.

syzymon committed:
Create hw1_st.ipynb

syzymon committed:
Update hw1_st.ipynb

pbiecek commented:
+add title
+add short comment to each trained model
+compare model performance, maybe with ROC curves

syzymon committed:
Add comments and roc curves

The title was already added in the previous commit. 
In this commit I added comments about each model performance and comparison, including ROC curves
